### DISCUSSION TITLE
Does the model run faster using Alpyne than simulating in anylogic?

### DISCUSSION THREAD

**Discussion by listen-codeing** [the-anylogic-company/Alpyne#57] [General]
I have a model to simulate the automated operation of a container terminal. 
Now I use the Pypeline to connect the Anylogic and Python, but I found that it runs too slow and I have to wait a long time to get the result. This means that I have to postpone my work. So if I use the Alpyne to connect my Anylogic model and Python, will it save me time?
I know that Alpyne can run many models with different parameters and I mean the general running speed for the simulation.

**Comment by t-wolfeadam** [comment]
I suppose I'm not clear how your scenario conflates Pypeline and Alpyne -- where Pypeline is used when you want to run Python code *inside* an AnyLogic model (i.e., to supplement your model logic), Alpyne is used for running an AnyLogic model *outside* of AnyLogic itself (i.e., from within a Python environment). In other words, they serve different purposes and one is not a replacement for the other. If you want to clarify your scenario, I'd be glad to answer further.
